### PR TITLE
Add Pipelines v0.59.x  release

### PR DESCRIPTION
**Pull request by khrm**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/kind misc

**Review by AlanGreene — APPROVED** [review]
Thanks @khrm 

v0.57 and v0.58 are EOL so they should be removed from the list.

File: sync/config/pipelines.yaml [hunk 1]
```diff
@@ -26,26 +26,16 @@ archive: https://github.com/tektoncd/pipeline/tags
...
+  displayName: v0.59.x-LTS
   # Dict of folders to sync
   folders:
     docs:
       index: README.md
       include: ['*.md']
       exclude:
         - api-spec.md
-        - tutorial.md 
-- name: release-v0.57.x
```
Comment (khrm) [inline sync/config/pipelines.yaml:39]: I have removed 0.57 and 0.58.

**Review by khrm — COMMENTED** [review]
v0.47.x and v0.44.x are also EOL. Should I remove them @AlanGreene ?

**Comment by AlanGreene** [comment]
Yes any other remaining EOL releases should be removed. That can be done in a separate PR, thanks.
/lgtm

**Pull request merged**